cves/kernel/CVE-2021-45095.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  It was discovered that the Phone Network protocol (PhoNet) implementation in the Linux kernel
+  did not properly perform reference counting in some error conditions. Phonet is a packet protocol 
```

Comment:
It could be a good idea to include a short definition of reference counting in this paragraph. Otherwise, very interesting!